README.md

```diff
@@ -14,7 +14,7 @@ To start contributing, you must clone this repository online, [GitHub Client (re
 
 ### World Version
 
--   Every map must work with Minecraft 1.8.9. To determine the version of a map, you can open the map's `level.dat` with a NBT editor and [find the value of `Data/DataVersion` or `Version`](https://minecraft.fandom.com/wiki/Data_version#List_of_data_versions).
+-   Every map must work with Minecraft 1.8.9. To determine the version of a map, you can open the map's `level.dat` with an NBT editor and [find the value of `Data/DataVersion` or `Version`](https://minecraft.fandom.com/wiki/Data_version#List_of_data_versions).
```

Comment:
Why change this? If I remember correctly, “a NBT editor” is grammatically correct

Reply:
Phonetically, "an" makes sense here as the most typical pronunciation of the term "NBT" from my understanding is such that the letters are said individually. Despite the first letter not being a vowel itself, the first sound of the term is a vowel sound.

Reply:
What @calcastor said is the reasoning behind my change.